src/docs/references/events/index.md

```diff
@@ -130,10 +96,90 @@ that have the exact same `createdAt` timestamp, so applications should
 not use the timestamp as a strict criteria to determining event
 ordering or to decide which events are processed and which are not.
 
+### Event data formats
+
+There are two data formats for events: **native** and **Integration API** event data formats.
+
+If the event is viewed via Flex CLI, the event and resource data is not normalized. Instead, it is inlined in a simplified form. We will refer to this as *native* event data format. On the other hand, when accessed via the Integration API, the event and associated resource data is formatted in the same way all Integration API resources are.
```

Comment:
@Gnito do you think it's ok to reverse the order in which formats are introduced here or somehow change then the description of the native format? The way it's presented now, it is defined in contrast with the Integration API format (i.e. "not normalized", "simplified"), which doesn't make sense as first thing before the Integration API format is explained. What was the issue you saw with the original sentence:

> when accessed via the Integration API, the event and associated
resource data is formatted in the same way all Integration API resources are
(with `id`, `type` top-level attributes and with `relationships` object
containing the IDs of related resources). On the other hand, if the event is
viewed via Flex CLI, the event and resource data is not normalized in that way
and is inlined in a simplified form.

Reply:
@bladealslayer No opinion about this. I just tried to combine different sentences when swapped the article structure. 
My suggestion is that if you agree with the structural change, you just merge this PR to yours and then rewrite the actual content under the headings if needed. :)